app/ante/ante_test.go

```diff
@@ -186,10 +185,20 @@ func (suite AnteTestSuite) TestAnteHandler() {
 				return tx
 			}, false, true, true,
 		},
+		{
```

Comment:
This is the only added test case here, the others updates are just settings the gas based on the MinimumGasPrice var.